lib/open_food_network/orders_and_fulfillments_report/customer_totals_report.rb

```diff
@@ -92,7 +94,9 @@ def rules
               proc { |_line_items| "" },
               proc { |_line_items| "" },
               proc { |_line_items| "" },
-              proc { |_line_items| "" }
+              proc { |_line_items| "" },
+              proc { |line_items| line_items.first.order.number },
+              proc { |line_items| line_items.first.order.completed_at.strftime("%F %T") },
```

Comment:
This approach to format the date looks ok, we have two more ways to render dates in reports:
- including timezone as in this to_local_time here:
https://github.com/openfoodfoundation/openfoodnetwork/blob/ed15bc422e92ea5cd8289f9d13b2a9c135709056/lib/open_food_network/users_and_enterprises_report.rb#L128
- and in Order and distributor report and sales tax report we are just using the date field directly: https://github.com/openfoodfoundation/openfoodnetwork/blob/ed15bc422e92ea5cd8289f9d13b2a9c135709056/lib/open_food_network/order_and_distributor_report.rb#L82 and https://github.com/openfoodfoundation/openfoodnetwork/blob/407d890d23c2001ad3060eec6633ab8bd1783ab3/lib/open_food_network/sales_tax_report.rb#L60

Changing the created_at to completed_at on this last sales_tax_report would give us a bonus issue #5306 :-)

Maybe we should fix all these in one go: make all use completed_at (all these reports look only at completed orders) and use this formatting you have introduced in this PR.

Reply:
I wonder if we should get rid of the timezone part in users_and_enterprises_report.rb or use it everywhere.

Reply:
> I wonder if we should get rid of the timezone part in users_and_enterprises_report.rb or use it everywhere.

My assumption would be that Rails parses datetimes into the local time zone anyway? The time zone is not actually being rendered in the output of `to_local_datetime`, so I don't think the `in_time_zone` has a big impact...?

I think we can probably use `order.completed_at.strftime("%F %T")` pretty much universally.

Reply:
cool, let's do that :+1: